src/Screens/MainPage/components/MainPage.tsx

```diff
@@ -14,26 +14,39 @@ import {
 
 import { dateData, sourcesData } from "../../../mockData";
 import Title from "../../../Comps/title/Title";
-import TitleStories from "../../../Comps/title/Title.stories";
+import DropDowns from "../../../Comps/dropdownsFilters/DropDowns";
+import { getLocation, getAllTopHeadlinesSources } from "../../../Services/Api";
+import { useDispatch, useSelector } from "react-redux";
+import { updateFilters } from "../../../store/query";
+import axios from "axios";
+import { RootState } from "../../../store/store";
+import { CountryData } from "../../../types/datatypes";
 
 const MainPage = () => {
+  const Query = useSelector((state: RootState) => state.query);
   const [dates, setDates] =
     useState<{ month: string; frequency: number }[]>(dateData);
   const [sources, setSources] =
     useState<{ name: string; value: number; fill: string }[]>(sourcesData);
 
+  const dispatch = useDispatch();
+
+  const regionNames = new Intl.DisplayNames(["en"], { type: "region" });
   useEffect(() => {
-    //call api and set all data and pass it to store of search query
-    //
+    getLocation().then((value) => {
+      dispatch(updateFilters({ country: value }));
+    });
   }, []);
 
   return (
     <>
       <Navbar />
       <Container>
-        {/* <DropDowns/> */}
+        <DropDowns />
         <UnderLine></UnderLine>
-        <Title subject="dfdf" city="dfsdf" />
+        <Title
+          city={regionNames.of(Query.query.filters.country.toUpperCase()) || ""}
```

Comment:
create a variable that will hold that values, it seems that u will use it in more places